create-presigned-download-url/src/main/resources/download_file.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Retrieve download URL for a Publication File
+
+  Scenario: A User retrieves a download URL for an Unpublished Publication File
+    Given the User is the owner of the Publication
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request GET /download/{identifier}/files/{fileIdentifier}/generate
```

Comment:
I think this line should be the first When. The scenario does not make much sense as it is written now.